test/test_core.py

```diff
@@ -9414,7 +9414,11 @@ def test_pthread_dylink_longjmp(self):
   @needs_dylink
   @node_pthreads
   def test_pthread_dylink_main_module_1(self):
-    self.emcc_args += ['-Wno-experimental', '-pthread', '-lhtml5']
+    # TODO: For some reason, -lhtml5 must be passed in -sSTRICT mode, but can NOT
+    # be passed when not compiling in -sSTRICT mode. That does not seem intentional?
```

Comment:
This certainly sounds like a bug. Could you open one an link it here (and below).